### ISSUE TITLE
Feat: Copying individual flashcards between decks

### ISSUE THREAD

**Issue by royalvarez** [asweigart/tsofa-the-simple-offline-flashcard-app#4]
Hello Mr. Sweigart,

I first want to express my appreciation for this wonderful light weight productivity tool. Many of the popular alternative online flashcards like Anki lack proper screen reader support. It is nice to find one that works out of the box.

I would like to suggest a feature regarding the ability to copy an individual flashcard which then can be pasted into a new deck, keeping its content. For example,
* I have deck 1 = card 1, card 2, card 3 and so on.
* I click a button on card 2, "copy flashcard".
* I can then go create a new deck, deck 2, and click the button, "paste flashcard."

Thank you sir!
